modules/apps/foundation/portal/portal-upload/src/main/java/com/liferay/portal/upload/configuration/UploadServletRequestConfiguration.java

```diff
@@ -29,10 +29,16 @@
 )
 public interface UploadServletRequestConfiguration {
 
-	@Meta.AD(deflt = "104857600", required = false)
+	@Meta.AD(
+		deflt = "104857600", description = "max-size-help",
+		name = "overall-maximum-upload-request-size", required = false
```

Comment:
Yeah, I'm not 100% clear what the rule should be. Maybe it's best to ask @brianchandotcom?